### DISCUSSION TITLE
Is there any specific reason for not implementing multiple values for a single property?

### DISCUSSION THREAD

**Discussion by anaamnizami** [flipt-io/flipt#584] [Q&A] [answered]
I was using flipt for A/B testing, and I noticed that you have to add same property multiple times, like in my case I wanted to give only 15 users access to a new feature, and I have to add 15 properties named email, to accommodate that. 

Is there any specific reason we can't have an array/list of users against same property like shown below. 
email in [abc, cde, ..... , xyz]

**Comment by markphelps** [comment, marked as answer]
> like in my case I wanted to give only 15 users access to a new feature, and I have to add 15 properties named email, to accommodate that.

Would you mind sharing either a screenshot or subset of the [export](https://flipt.io/docs/configuration/#export) that contains the feature/fields in question?

I'm wondering if there isn't a simpler way to accomplish what you are trying to do

**Reply by anaamnizami** [reply]
hey mark, screenshot of the feature in my application or screenshot of constraints I am setting in flipt?

**Reply by markphelps** [reply]
hey, sorry. screenshots of the constraints in Flipt. Im trying to understand how you are modeling it in Flipt, so I can better understand the issue
